tests/SearchBarFixture.js

```diff
@@ -0,0 +1,3 @@
+aahQuery = "aah"
+aahResultName = "aah"
+aahResultDescription = "Allocation adulte handicapé (Individu) mensualisée"
```

Comment:
Too much details, this makes us sensitive to this (bad) description. `/Allocation adulte handicapé/` should be enough.